doc/classes/Image.xml

```diff
@@ -351,24 +376,28 @@
 			<argument index="2" name="color" type="Color">
 			</argument>
 			<description>
+				Colors in a pixel at position [code](x, y)[/code] with [code]color[/code].
```

Comment:
- "Colors in a pixel" sounds like it's returning the color, not setting it
- Link to [Color] (user might not know the right format)

Sets the [Color] of the pixel at [code](x, y)[/code].

Reply:
It might be worth mentioning that image has to be unlocked in order to write to it, maybe also a little example.